ivy/functional/frontends/numpy/random/functions.py

```diff
@@ -197,3 +197,11 @@ def logistic(loc=0.0, scale=1.0, size=None):
     u = ivy.random_uniform(low=0.0, high=0.0, shape=size, dtype="float64")
     x = loc + scale * ivy.log(u / (1 - u))
     return x
+
+
+@to_ivy_arrays_and_back
+@from_zero_dim_arrays_to_scalar
+def laplace(loc=0.0, scale=1.0, size=None):
+    u = ivy.random_uniform(low=0.0, high=1.0, shape=size, dtype="float64")
+    x = ivy.exp(-abs(u - loc) / scale)/ (2 * scale)
```

Comment:
This implementation is incorrect, I believe the correct implementation would look something like this:
```
    u = ivy.random_uniform(low=-0.5, high=0.5, shape=size)
    x = loc - (scale * ivy.sign(u) * ivy.log(1 - (2 * ivy.abs(u))))
```
Try sampling the frontend function to see the difference, and compare the results to the native numpy function.

Reply:
Hey @Sam-Armstrong ,

Thank you so much for pointing it out. This function is closest to Numpy's laplace function. I tried using the code mentioned below, it also works fine, but yours seems more intuitive. 

`def laplace(loc=-0.0, scale=1.0, size=None):
<img width="719" alt="Screenshot 2023-08-12 at 1 17 14 PM" src="https://github.com/unifyai/ivy/assets/54806252/3b88cffa-ac0a-4c3f-bef3-a33035012dba">

    u = ivy.random_uniform(low=0.0, high=1.0, shape=size)
    u_normalized = u - 0.5
    x = loc - (scale * ivy.sign(u_normalized) * ivy.log(1 - (2 * ivy.abs(u_normalized))))
    return x
`
I tried s
<img width="823" alt="Screenshot 2023-08-12 at 1 17 46 PM" src="https://github.com/unifyai/ivy/assets/54806252/bb722e0c-e10a-4b14-a127-e1b21130246b">
sampling both the functions and compared it with numpy's. I have attached the screenshot.

I have made the changes, but the commits got deleted due to conflicts. I had to create a [new PR](https://github.com/unifyai/ivy/pull/21747). Please review it and let me know your suggestions.